server/game/cards/characters/05/lannisportguard.js

```diff
@@ -0,0 +1,19 @@
+const DrawCard = require('../../../drawcard.js');
+
+class LannisportGuard extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardEntersPlay: (e, card) => card === this && this.game.currentPhase === 'marshal'
+            },
+            handler: () => {
+                player.drawCardsToHand(1);
```

Comment:
`player` doesn't exist here, and the card ability is for each player to draw a card. So what you really want here is an each loop through each player (which will define a `player` variable for you to use), then draw the card inside that loop.
```javascript
_.each(this.game.getPlayers(), player => {
    player.drawCardsToHand(1);
});
```

You'll also need to pull in the Underscore object at the top of the file:
```
const _ = require('underscore');
```